guides/17_Migration_to_3_0.md

```diff
@@ -0,0 +1,125 @@
+# Migration from SDK v1 to SDK v2
+
+## Installation
+
+Installation involves adjusting your `package.json` to depend on version `3.0.0`.
+
+```shell
+npm install --save amazon-chime-sdk-js@3
+```
+
+## Interface changes
+
+Version 3 of the Amazon Chime SDK for JavaScript makes a number of interface
+changes.
+
+In many cases you should not need to adjust your application code at all. This will be the case if:
```

Comment:
Love this section. I wouldn't even mind putting it in bold.

Reply:
It seems a bit much to bold the entire section, how about just the first two lines?